webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -40,6 +42,7 @@ export function combineReducersAsync(asyncReducers) {
     ...factChartReducers,
     ...intervalReducers,
 
+    router: connectRouter(history),
```

Comment:
Wondering if we should follow the convention and create a `react_app/routes/RouterReducer.js` with:
```js
import { connectRouter } from 'connected-react-router';
import history from '../../history';

export const routerReducers = {
  router: connectRouter(history),
};
```

So in the reducers file you would do `...routerReducers`

Reply:
Looks overkill to me